docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -1823,7 +1823,7 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
 
   ```diff
   -import { StylesProvider } from '@material-ui/core/styles';
-  +import { StyledEngineProvider } from '@material-ui/styles';
+  +import { StylesProvider } from '@material-ui/styles';
```

Comment:
I'm still not sure about this part. 
The patch above already shows that `StylesProvider` has been changed to `StyledEngineProvider`, so I don't understand why this part of the document still remains the same? And while I was working on my project, it's impossible to `import { StylesProvider } from '@material-ui/styles'`

Reply:
> The patch above already shows that StylesProvider has been changed to StyledEngineProvider

These are two different modules.

Reply:
> No, these are two different modules and both still exist.

Oh, all rights. I thought they were the same.

> Because you don't have the right package installed?

Not sure though, I'm using like: `"@material-ui/core": "next"` in `package.json`